package.json

```diff
@@ -10,10 +10,13 @@
     "IE 11",
     "not dead"
   ],
+  "optionalDependenciesComments": {
+    "MoJ Frontend": "upload component version 3.x has broken styling with ;"
+  },
   "dependencies": {
     "@hotwired/stimulus": "^3.2.2",
     "moment": "^2.30.1",
-    "@ministryofjustice/frontend": "^3.7.0",
+    "@ministryofjustice/frontend": "^2.2.5",
```

Comment:
Should we avoid `dependabot` trying to upgrade it by adding it to the `ignore` section for `npm` ecosystem in the `.github/dependabot.yml` file?

Reply:
v3.7.1 has a bug fix for it, but it doesn't quite work. TBD...